csrc/fusion_segmenter.h

```diff
@@ -124,6 +124,19 @@ class SegmentedGroup {
     return exprs_;
   }
 
+  //! Returns the complete fusion inputs mapped to this segmented group's fusion
+  const auto& getCompleteFusionInputs() const {
```

Comment:
Note to myself: complete fusion inputs are used by FusionKernelRuntime::getMaybeHeuristicsFor, which holds a PrecomputedValues for the complete fusion. FusionKernelRuntime::getMaybeHeuristicsFor is on the critical path when not making the initial heuristics.